frontend/src/components/RegisterForm.jsx

```diff
@@ -22,7 +22,7 @@ const RegisterForm = (props) => {
             <Input prefix={<UserOutlined className="site-form-item-icon" />} placeholder="Username" />
           </Form.Item>
           <Form.Item
-            name="github_url"
+            name="git_url"
```

Comment:
I personally dont recommend keep `github_url `, `git_url` includes platforms like `gitlab`, in our case, be specific, use `github_url`